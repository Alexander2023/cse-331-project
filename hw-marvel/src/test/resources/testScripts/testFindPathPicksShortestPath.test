# Tests that path finding applies BFS properly by choosing the shorter of two paths

CreateGraph g

AddNode g c1
AddNode g c2
AddNode g c3

AddEdge g c1 c2 b1
AddEdge g c2 c3 b2
AddEdge g c1 c3 b3

FindPath g c1 c3