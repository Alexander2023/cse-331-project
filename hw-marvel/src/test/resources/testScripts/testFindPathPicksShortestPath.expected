# Tests that path finding applies BFS properly by choosing the shorter of two paths

created graph g

added node c1 to g
added node c2 to g
added node c3 to g

added edge b1 from c1 to c2 in g
added edge b2 from c2 to c3 in g
added edge b3 from c1 to c3 in g

path from c1 to c3:
c1 to c3 via b3